### ISSUE TITLE
mods villager professions conflict 

### ISSUE THREAD

**Issue by mser7** [Daveyx0/PrimitiveMobs#205]
...again
![2019-01-17_10 04 37](https://user-images.githubusercontent.com/22555489/51305248-ead30980-1a6c-11e9-83d5-24e0c5b3bb50.png)
![2019-01-17_10 04 40](https://user-images.githubusercontent.com/22555489/51305249-eb6ba000-1a6c-11e9-9031-4da4d91cd1dc.png)

**Comment by mser7** [comment]
![2019-01-29_10 37 54](https://user-images.githubusercontent.com/22555489/51976607-ad806a00-24b7-11e9-8d14-b6dfec0b41ba.png)

**Comment by Daveyx0** [comment]
Really strange. Did you find the issue in the end?

**Comment by mser7** [comment]
Nope, I closed it because I realized I am using an outdated version and can't report bugs, due to the spawning system of 1.2.3a

**Comment by mser7** [comment]
Nope, tested in 1.2.3a and now I am going to downgrade from that version

**Comment by mser7** [comment]
closing again, apparently this is a issue of Just Enough Resources
![2019-02-10_16 12 48](https://user-images.githubusercontent.com/22555489/52551909-fa960180-2e10-11e9-85ac-146ee6ed8051.png)

**Comment by mser7** [comment]
Nope, my mistake
